pkgs/servers/pufferpanel/disable-group-checks.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/cmd/main.go b/cmd/main.go
+index f9af7038..099ff2e2 100644
+--- a/cmd/main.go
++++ b/cmd/main.go
+@@ -24,11 +24,6 @@ import (
+ )
+ 
+ func main() {
+-	if !pufferpanel.UserInGroup("pufferpanel") {
+-		fmt.Println("You do not have permission to use this command")
+-		return
+-	}
+-
+ 	defer logging.Close()
+ 
+ 	defer func() {
+diff --git a/cmd/user.go b/cmd/user.go
+index d4a27aaf..d633694b 100644
+--- a/cmd/user.go
++++ b/cmd/user.go
+@@ -17,7 +17,6 @@ import (
+ 	"errors"
+ 	"fmt"
+ 	"github.com/AlecAivazis/survey/v2"
+-	"github.com/pufferpanel/pufferpanel/v2"
```

Comment:
This is worse for merge conflicts with future updates in case they start using this import elsewhere.

Reply:
Done, changed the patch to `_ = pufferpanel.UserInGroup` to keep the import.